app/src/containers/ConnectedSidebar/tests/index.test.tsx

```diff
@@ -45,15 +49,15 @@ describe('components/ConnectedSidebar', () => {
         </MemoryRouter>
       </Provider>
     );
-    expect(wrapper.find('Menu').at(0).prop('children')).toMatchSnapshot();
-    wrapper.unmount();
+    expect(wrapper.find('aside')).toHaveLength(1);
   });
 
-  it('displays menu links for enabled modules', () => {
+  it('Test order of menu', () => {
     const envModule = require('../../../configs/env');
-    envModule.ENABLE_FORM_CONFIGURATION = 'true';
     envModule.ENABLE_PRODUCT_CATALOGUE = 'true';
     envModule.ENABLE_PLANS = 'true';
+    envModule.ENABLE_LOCATIONS = 'true';
+    envModule.ENABLE_FORM_CONFIGURATION = 'true';
     envModule.ENABLE_CARD_SUPPORT = 'true';
```

Comment:
Should we add `ENABLE_INVENTORY` and `ENABLE_TEAMS` too?

Reply:
yes we should by that time i think we didn't had those that's why i didnt added them